### PR TITLE
WIP: SC Updates

### PR DESCRIPTION
**Pull request by ghalym**
This PR includes the changes for the SC
1. Updated the interface to all the lfe PLCs with the latest st beam parameters.
2.  Added the FB BC to BCD update the asserted beam to the MPS crate.
3. Updated Current beam to include eV  and current beam class from PV.

### CODE REVIEW

File: Arbiter/ArbiterPLC/POUs/PMPS_Arbiter.TcPOU [hunk 1]
```diff
@@ -101,11 +104,11 @@ END_VAR
...
 // Updating arbiter with subsystem requests.
-plc_lfe_motion_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset);
-plc_lfe_optics_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset);
-plc_lfe_vac_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset);
-plc_lfe_gatt_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset);
-plc_xrt_homs_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset);
+plc_lfe_motion_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset, sName:= 'LFE MOTION');
+plc_lfe_optics_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset , sName:= 'LFE OPTICS');
+plc_lfe_vac_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset, sName:= 'LFE VACUUM');
+plc_lfe_gatt_PMPS_IO(Arbiter:=GVL.g_fbArbiter1, fbFFHWO:=GVL.g_FastFaultOutput1, Reset:=GVL.AttemptReset, sName:= 'LFE GEMS');
```
Comment (nrwslac) [inline Arbiter/ArbiterPLC/POUs/PMPS_Arbiter.TcPOU:110]: is gatt and gem the same?

**Comment by nrwslac** [comment]
Looks good to me.

**Review by nrwslac — APPROVED** [review]

File: Arbiter/ArbiterPLC/POUs/PMPS_Arbiter.TcPOU (hunk 1 above)
Comment (nrwslac) [reply Arbiter/ArbiterPLC/POUs/PMPS_Arbiter.TcPOU:110]: I see.

File: Arbiter/ArbiterPLC/POUs/Fast_Faults_Evaluate.TcPOU [hunk 2]
```diff
@@ -7,7 +7,11 @@ END_VAR
 ]]></Declaration>
     <Implementation>
       <ST><![CDATA[//g_FastFaultOutput1.EvaluateOutput(bAutoReset:=TRUE);
-//g_FastFaultOutput2.EvaluateOutput(bAutoReset:=TRUE);]]></ST>
+//g_FastFaultOutput2.EvaluateOutput(bAutoReset:=TRUE);
+
+// FF
+g_FastFaultOutput1.Execute();
+g_FastFaultOutput2.Execute();]]></ST>
```
Comment (ZLLentz) [inline Arbiter/ArbiterPLC/POUs/Fast_Faults_Evaluate.TcPOU:14]: Kind of scary that this was missing

**Review by ZLLentz — APPROVED** [review]
Everything here looks good to me!

File: Arbiter/ArbiterPLC/POUs/Fast_Faults_Evaluate.TcPOU (hunk 2 above)
Comment (ghalym) [reply Arbiter/ArbiterPLC/POUs/Fast_Faults_Evaluate.TcPOU:14]: It wasn't missing. It was running on the main (slow) task. And we had an FFO task with nothing running on it.
So, I moved it to this task.

Comment (ZLLentz) [reply Arbiter/ArbiterPLC/POUs/Fast_Faults_Evaluate.TcPOU:14]: Makes sense to me!

**Pull request merged**